web/src/components/Storyblok/StoryblokTestimonialFullWidth/StoryblokTestimonialFullWidth.tsx

```diff
@@ -0,0 +1,45 @@
+import { TestimonialFullWidthStoryblok } from "@/types/storyblok";
+
+import { Testimonial } from "../../Testimonial/Testimonial";
+import { StoryblokImage } from "../StoryblokImage/StoryblokImage";
+import { StoryblokRelatedLink } from "../StoryblokRelatedLink/StoryblokRelatedLink";
+
+export interface StoryblokTestimonialFullWidthProps {
+	blok: TestimonialFullWidthStoryblok;
+	className?: string;
+}
+
+export const StoryblokTestimonialFullWidth: React.FC<
+	StoryblokTestimonialFullWidthProps
+> = ({ blok, className = undefined }) => {
+	const {
+		variant,
+		quoteText,
+		quoteName,
+		quoteRole,
+		children,
+		headingLevel = 4,
```

Comment:
Do we need these? We'd discussed maybe removing these as we didn't think they were being used

Same on TestimonialGridItem